modules/programs/contour.nix

```
@@ -0,0 +1,1193 @@
+            default = [ ];
+            example = ''
+              ["Control" "Alt"]
+            '';
+            description = ''
+              Modifier keys.
+            '';
+          };
+
+          key = mkOption {
+            type = types.nullOr types.str;
+            default = null;
+            example = ''
+              Space
+            '';
+            description = ''
+              Keys can be expressed case-insensitively symbolic:
+
+              APOSTROPHE, ADD, BACKSLASH, COMMA, DECIMAL, DIVIDE, EQUAL, LEFT_BRACKET,
+              MINUS, MULTIPLY, PERIOD, RIGHT_BRACKET, SEMICOLON, SLASH, SUBTRACT, SPACE
+              Enter, Backspace, Tab, Escape, F1, F2, F3, F4, F5, F6, F7, F8, F9, F10, F11, F12,
+              DownArrow, LeftArrow, RightArrow, UpArrow, Insert, Delete, Home, End, PageUp, PageDown,
+              Numpad_NumLock, Numpad_Divide, Numpad_Multiply, Numpad_Subtract, Numpad_CapsLock,
+              Numpad_Add, Numpad_Decimal, Numpad_Enter, Numpad_Equal,
+              Numpad_0, Numpad_1, Numpad_2, Numpad_3, Numpad_4,
+              Numpad_5, Numpad_6, Numpad_7, Numpad_8, Numpad_9.
+
+              In case of standard characters, just the character.
+            '';
+          };
+
+          mouse = mkOption {
+            type = types.nullOr
+              (types.enum [ "Left" "Middle" "Right" "WheelUp" "WheelDown" ]);
+            default = null;
+            description = ''
+              Mouse button specification.
+            '';
+          };
+
+          action = mkOption {
+            type = types.enum [
+              "CancelSelection"
+              "ChangeProfile"
+              "ClearHistoryAndReset"
+              "CopyPreviousMarkRange"
+              "CopySelection"
+              "DecreaseFontSize"
+              "DecreaseOpacity"
+              "FollowHyperlink"
+              "IncreaseFontSize"
+              "IncreaseOpacity"
+              "NewTerminal"
+              "OpenConfiguration"
+              "OpenFileManager"
+              "PasteClipboard"
+              "PasteSelection"
+              "Quit"
+              "ReloadConfig"
+              "ResetConfig"
+              "ResetFontSize"
+              "ScreenshotVT"
+              "ScrollDown"
+              "ScrollMarkDown"
+              "ScrollMarkUp"
+              "ScrollOneDown"
+              "ScrollOneUp"
+              "ScrollPageDown"
+              "ScrollPageUp"
+              "ScrollToBottom"
+              "ScrollToTop"
+              "ScrollUp"
+              "SendChars"
+              "ToggleAllKeyMaps"
+              "ToggleFullScreen"
+              "ToggleTitleBar"
+              "ViNormalMode"
+              "WriteScreen"
+            ];
+            description = ''
+              Action to be executed on mapped input activation.
+            '';
+          };
+
+          mode = mkOption {
+            type = types.nullOr types.str;
+            default = null;
+            description = lib.mdDoc ''
```

Need to use docbook. HM doesn't use the documentation system of NixOS.

```suggestion
            description = ''
```